sympy/printing/lambdarepr.py

```diff
@@ -348,6 +348,13 @@ def _print_Float(self, e):
         args = str(tuple(map(int, e._mpf_)))
         return 'mpf(%s)' % args
 
+    def _print_uppergamma(self,e): #printer for the uppergamma function
+        exp="gammainc({0}, {1}, inf)".format(e.args[0], e.args[1])
+        return "(%s)" %(self._print(exp))
```

Comment:
I mean a single *formatting* operation like in 
`return "gammainc({0}, {1}, inf)".format(self._print(e.args[0]), ...`

Compare with e.g. [this line](https://github.com/sympy/sympy/blob/master/sympy/printing/lambdarepr.py#L198) (though preferably with .format instead of %).